From af57061033cc35c6b5e48fbad12610251dd0da20 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 10 Apr 2016 16:57:18 +0900
Subject: [PATCH 1/2] Red Hat

The following section mention Red Hat.  So skipping here is very awkward.
---
 en-US/00b_foreword.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/en-US/00b_foreword.xml b/en-US/00b_foreword.xml
index bdce254..a4ef8e1 100644
--- a/en-US/00b_foreword.xml
+++ b/en-US/00b_foreword.xml
@@ -38,7 +38,8 @@
       <para>Most Linux distributions are backed by a for-profit company
       that develops them and sells them under some kind of commercial
       scheme. Examples include <emphasis>Ubuntu</emphasis>, mainly
-      developed by <emphasis>Canonical Ltd.</emphasis>; <emphasis>Mandriva
+      developed by <emphasis>Canonical Ltd.</emphasis>; <emphasis>Red Hat
+      Enterprise Linux</emphasis>, by <emphasis>Red Hat</emphasis>; <emphasis>Mandriva
       Linux</emphasis>, by French company <emphasis>Mandriva SA</emphasis>;
       and <emphasis>Suse Linux</emphasis>, maintained and made commercially
       available by <emphasis>Novell</emphasis>.</para>
-- 
2.8.0.rc3

